bindings/python/Makefile.am

```diff
@@ -33,7 +33,7 @@ EXTRA_DIST = \
 if ENABLE_PYTHON_BINDING
 
 all-local:
-	$(PYTHON) $(srcdir)/setup.py build
+	CXXFLAGS="$(CXXFLAGS)" $(PYTHON) $(srcdir)/setup.py build
```

Comment:
I guess not, I can't find anything. However, I think it would be better to forward the result of ``AX_CXX_COMPILE_STDCXX_11([noext], [mandatory])`` instead.